merlion/models/automl/autoets.py

```diff
@@ -30,3 +92,166 @@ class AutoETS(SeasonalityLayer):
+            S_range = ["add", None]
+        else:
+            S_range = ["add", "mul", None]
+        D_range = [True, False]
+
+        if not self.config.auto_error:
+            E_range = [self.model.config.error]
+        if not self.config.auto_trend:
+            T_range = [self.model.config.trend]
+        if not self.config.auto_seasonal:
+            S_range = [self.model.config.seasonal]
+        if not self.config.auto_damped:
+            D_range = [self.model.config.damped_trend]
+
+        thetas = []
+        for error, trend, seasonal, damped in product(E_range, T_range, S_range, D_range):
+            if trend is None and damped:
+                continue
+            # if (error, trend, seasonal, damped) not in variants_pool_pred_interval:
+            #     continue
+            if self.config.additive_only:
+                if error == "mul" or trend == "mul" or seasonal == "mul":
+                    continue
+            if self.config.restrict:
+                if error == "add" and (trend == "mul" or seasonal == "mul"):
+                    continue
+                if error == "mul" and trend == "mul" and seasonal == "add":
+                    continue
+
+            thetas.append([error, trend, seasonal, damped])
+        return iter(thetas)
+
+    def evaluate_theta(
+        self, thetas: Iterator, train_data: TimeSeries, train_config=None, **kwargs
+    ) -> Tuple[Any, Optional[ETS], Optional[Tuple[TimeSeries, Optional[TimeSeries]]]]:
+        y = train_data.to_pd()[self.target_name]
+        for error, trend, seasonal, damped in thetas:
+            start = time.time()
+            _model_fit = _fit_ets_model(self.config.seasonal_periods, error, trend, seasonal, damped, y)
+            fit_time = time.time() - start
+            ic = getattr(_model_fit, self.config.information_criterion)
+            logger.debug(
+                "{model}   : {ic_name}={ic:.3f}, Time={time:.2f} sec".format(
+                    model=_model_name(_model_fit.model), ic_name=self.config.information_criterion.upper(), ic=ic,
+                    time=fit_time
+                )
+            )
+            self.config._results_dict[(error, trend, seasonal, damped)] = _model_fit
+            self.config._ic_dict[(error, trend, seasonal, damped)] = ic
+            if self.config._bestfit is None:
+                self.config._bestfit = _model_fit
+                self.config._bestfit_key = (error, trend, seasonal, damped)
+                logger.debug("First best model found (%.3f)" % ic)
+            current_ic = self.config._ic_dict[self.config._bestfit_key]
+            if ic < current_ic:
+                logger.debug("New best model found (%.3f < %.3f)" % (ic, current_ic))
+                self.config._bestfit = _model_fit
+                self.config._bestfit_key = (error, trend, seasonal, damped)
+
+
+        best_model_theta =  self.config._bestfit_key + (self.config.seasonal_periods,) + (self.config.pred_interval_strategy,)
```

Comment:
Why include `pred_interval_strategy` as a part of `theta`? It seems like this is a fixed parameter.